CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # edge-react-gui
 
+## 2.0.14 (2021-06-24)
+
+- Show FIO renewal reminders
```

Comment:
This should be removed and changed with:
`Only allow for one FIO expired pop-up per login `